app/services/nomis/elite2/user_api.rb

```diff
@@ -0,0 +1,13 @@
+module Nomis
+  module Elite2
+    class UserApi
+      extend Elite2Api
+
+      def self.fetch_email_addresses(nomis_staff_id)
+        route = "/elite2api/api/staff/#{nomis_staff_id}/emails"
+        emails = e2_client.get(route)
+        emails
```

Comment:
The assignment is unnecessary, you can remove emails and emails= ... it'll return the last thing in the function so just ```
route = 
e2_client.get...
```
should be enough